packages/dev/parcel-transformer-mdx-docs/MDXTransformer.js

```diff
@@ -441,26 +451,25 @@ function transformExample(node, preRelease) {
       },
       Program: {
         exit(path) {
-          if (specifiers.length > 0) {
-            let literal =  t.stringLiteral('@adobe/react-spectrum');
-            literal.raw = "'@adobe/react-spectrum'";
-
-            let decl = t.importDeclaration(
-              specifiers.map(s => t.importSpecifier(t.identifier(s), t.identifier(s))),
-              literal
-            );
-
-            decl.loc = last.loc;
-            decl.start = last.start;
-            decl.end = last.end;
-
-            path.unshiftContainer('body', [decl]);
+          for (let lib in specifiers) {
+            let names = specifiers[lib];
+            if (names.length > 0) {
+              let monopackage = lib === '@react-spectrum' ? '@adobe/react-spectrum' : lib.slice(1);
+              let literal =  t.stringLiteral(monopackage);
+
+              let decl = t.importDeclaration(
+                names.map(s => t.importSpecifier(t.identifier(s), t.identifier(s))),
+                literal
+              );
+
+              path.unshiftContainer('body', [decl]);
+            }
           }
         }
       }
     });
 
-    node.value = generate(ast).code.replace(/(<WRAPPER>(?:.|\n)*<\/WRAPPER>)/g, '\n($1)');
+    node.value = recast.print(ast, {objectCurlySpacing: false, quote: 'single'}).code.replace(/(<WRAPPER>(?:.|\n)*<\/WRAPPER>)/g, '\n($1)');
```

Comment:
Recast does a better job of code generation, because it preserves the original formatting for anything that wasn't modified.